shared/src/libs/validation-schemas/find-by-query-entries.validation-schema.ts

```diff
@@ -0,0 +1,9 @@
+import joi from 'joi';
+
+import { type GetItemsByQueryDto } from '#index.js';
+
+const findByQueryEntries = joi.object<GetItemsByQueryDto, true>({
```

Comment:
```suggestion
const journalEntriesFiltering = joi.object<GetItemsByQueryDto, true>({
```

Reply:
@Liza-Veis @v0ldemar01 told me to make one schema for both journal and chats in this [comment](@Liza-Veis @v0ldemar01 told me to make one )

Reply:
the same as above